src_cpp/bp.hpp

```diff
@@ -80,7 +82,7 @@ typedef ldpc::gf2sparse::GF2Sparse<BpEntry> BpSparse;
                 double min_sum_scaling_factor = 0.625,
                 int omp_threads = 1,
                 std::vector<int> serial_schedule = NULL_INT_VECTOR,
-                int random_schedule_seed = 0,
+                int random_schedule_seed = -1, // TODO what should be default here? 0 is set but -1 is checked in decode method?
```

Comment:
If it is -1, then you have to manually specify the schedule.
For seed values `>=0`, the schedule is randomly set.